src/main/resources/templates/cart.html

```diff
@@ -59,10 +51,12 @@ <h1>我的购物车</h1>
                 <td><img th:src="${sItem.getCurrentItem().getImage()}"></td>
                 <td th:text="${sItem.getCurrentItem().getDescr()}"></td>
                 <td class="sPrice" th:text="${sItem.getCurrentItem().getPrice()}"></td>
-                <td >
-                    <input type="hidden" name="sItemID" id="sItemID" th:value="${sItem.getCurrentItem().getId()}">
-                    <label for="sItemNo">当前数量</label>
-                    <input class="ItemNo" type="number" name="sItemNo" id="sItemNo" th:value="${sItem.getItemNum()}">
+                <td>
+                    <form class="sItemCartUpdateForm">
+                        <input type="hidden" name="sItemID" id="sItemID" th:value="${sItem.getCurrentItem().getId()}">
+                        <label for="sItemNo">当前数量</label>
+                        <input class="ItemNo" type="number" name="sItemNo" id="sItemNo" th:value="${sItem.getItemNum()}">
+                    </form>
```

Comment:
Please double check the logic in your javascript, after you made modification on this place, to make sure it works.

Reply:
you mean when i change item num from front page, should Synchronize to the database immediately?

Reply:
i designed when you change num, you shoud click updatecart button to synchronize to the database